### PR TITLE
fix(acceptancetests): add teamVersion

### PR DESCRIPTION
**Pull request by jstrachan**
lets add a team version too so we can configure that in addition to jenkins & che versions in the acceptance tests in PRs

### CODE REVIEW

**Comment by fabric8cd** [comment]
@jstrachan fabric8/fabric8-ui:SNAPSHOT-PR-1913-1 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1913-fabric8-ui.badger.fabric8.io

File: src/app/profile/tenant/tenant.component.html [hunk 1]
```diff
@@ -42,6 +42,14 @@
           </div>
         </div>
         <div class="form-group">
+          <label for="teamVersion" class="control-label">Team version</label>
```
Comment (joshuawilson) [inline src/app/profile/tenant/tenant.component.html]: @jstrachan where does this text appear in the UI?

Comment (jstrachan) [reply src/app/profile/tenant/tenant.component.html]: on  the advanced OSIO developer/tester page: https://openshift.io/my-user-name/_tenant - its invisible to everyone who doesn't know that magic URL

one day we may even make that magic URL invisible to non 'OSIO testers'

Comment (joshuawilson) [reply src/app/profile/tenant/tenant.component.html]: ah, thanks

Comment (jstrachan) [reply src/app/profile/tenant/tenant.component.html]: background to this 'hidden' page here: https://github.com/fabric8-services/fabric8-tenant#testing-yaml

**Pull request merged**